fusion-cli/build/plugins/instrumented-import-dependency-template-plugin.js

```diff
@@ -108,19 +115,19 @@ InstrumentedImportDependency.Template = class InstrumentedImportDependencyTempla
     } else if (this.clientChunkIndex) {
       // Template invoked without InstrumentedImportDependency
       // server-side, use values from client bundle
-      let ids = this.clientChunkIndex.get(
-        (dep.module && dep.module.resource) || dep.originModule.resource
+      const ids = this.clientChunkIndex.get(
+        dep.module instanceof ConcatenatedModule
```

Comment:
With the translations instrumentation, the chunk ids got out of sync between server and client build. This caused the i18n node plugin to lookup chunk ids that apparently had no translations.

The fix here is to use `dep.module.rootModule` for `ConcatenatedModules` instead of `dep.originModule` because the latter refers to the *importing* module, whereas the former refers to the *imported* module.